src/app/public/modules/inline-delete/inline-delete.component.html

```diff
@@ -0,0 +1,41 @@
+<div
+  aria-describedby="sky-inline-delete-assistive-text"
+  class="sky-inline-delete sky-inline-delete-{{type}}"
+  role="alertdialog"
+  [attr.aria-label]="'skyux_inline_delete_confirm_deletion' | skyLibResources"
+>
+  <span
+   class="sky-inline-delete-assistive-text"
+   id="sky-inline-delete-assistive-text"
+  >
+    {{ 'skyux_inline_delete_assistive_text' | skyLibResources }}
+  </span>
+  <div *ngIf="pending"
+    class="sky-inline-delete-wait"
+  >
+    <sky-wait
+      [isWaiting]="true"
+    ></sky-wait>
+  </div>
+  <div
+    class="sky-inline-delete-content"
+    [@skyAnimationEmerge]="animationState"
+    (@skyAnimationEmerge.done)="onAnimationDone($event)"
+  >
+    <button
+      autofocus
+      class="sky-btn sky-btn-danger sky-inline-delete-button"
+      type="button"
+      (click)="deleteTriggered.emit()"
+    >
+      {{ 'skyux_inline_delete_delete' | skyLibResources }}
+    </button>
+    <button
+      class="sky-btn sky-btn-default"
+      type="button"
+      (click)="animationState = 'closed'"
+    >
+      {{ 'skyux_inline_delete_cancel' | skyLibResources }}
+    </button>
+  </div>
+  </div>
```

Comment:
Is the overall tabbing off on this `div`? Seems like its missing one tab.

Reply:
I think the closing div tag for the main div was just tabbed over one too many times.

Reply:
Done